include/hyprcursor/hyprcursor.hpp

```diff
@@ -68,6 +68,7 @@ namespace Hyprcursor {
             \since 0.1.6
         */
         CHyprcursorManager(const char* themeName, PHYPRCURSORLOGFUNC fn);
+        CHyprcursorManager(const char* themeName, PHYPRCURSORLOGFUNC fn, bool allowDefaultFallback);
```

Comment:
at this point I'd make a `struct SManagerOptions`

Reply:
do you want all of them to use SManagerOptions or just the new one?

Reply:
tell me if that works for you